mlrun/api/schemas/clusterization_spec.py

```diff
@@ -0,0 +1,14 @@
+import enum
+import typing
+
+import pydantic
+
+
+class ClusterizationSpec(pydantic.BaseModel):
+    chief_api_state: str
```

Comment:
You're setting it with `mlrun.mlconf.httpdb.state if is_chief else None` so it needs to optional